src/view/com/modals/ContentFilteringSettings.tsx

```diff
@@ -188,7 +192,7 @@ function SelectGroup({current, onChange, group}: SelectGroupProps) {
       />
       <SelectableBtn
         current={current}
-        value="show"
+        value="ignore"
         label="Show"
```

Comment:
label is still `show` but value is `ignore`

Reply:
This is intentional. The internal logic is to ignore the label, but I figure the clearest move is to have the UI say "Show". Can revisit, but not just a missed change.

Reply:
Gothca!